src/open_inwoner/qmatic/client.py

```diff
@@ -135,10 +136,16 @@ def list_appointments_for_customer(
         if response.status_code == 404:
             return []
         response.raise_for_status()
+        config = QmaticConfig.get_solo()
         try:
-            return [
+            appointments = [
                 Appointment(**entry) for entry in response.json()["appointmentList"]
             ]
+            for appointment in appointments:
+                appointment.url = (
+                    f"{config.booking_base_url}{quote(appointment.publicId)}"
```

Comment:
Turns out I can't actually use `furl` here, because the URL Hoorn uses has `/#/` in its path, which `furl` parses as the fragment and moves it to the end of the URL

Reply:
Ugh.. seems fishy but if it works :man_shrugging: